### DISCUSSION TITLE
为翻译文献而生，饺子划词翻译助手+沙拉查词【Mac+Windows】

### DISCUSSION THREAD

**Discussion by zwjjiaozhu** [crimx/ext-saladict#1828] [Show and tell]
为翻译文献而生，饺子划词助手+沙拉查词【Mac+Windows】

[TOC]

# 介绍

[饺子翻译助手](https://study.zwjjiaozhu.top)是西园公子原创的一款划词翻译助手，支持在**主流PDF阅读器**中划词选词，当鼠标划词后会调用专业级翻译软件，进而**立即出现翻译结果**。这优化了传统方法中需要二次确认才可翻译的问题，实现划词后**秒翻译**。
![Acrobat+饺子翻译助手+沙拉查词](https://user-images.githubusercontent.com/34623422/187384946-a96600af-9889-4212-9629-90dcf8889e97.png)

依托知名的轻量级翻译插件：[沙拉查词](https://saladict.crimx.com/)、划词翻译、[可可翻译](https://github.com/chunibyocola/sc-translator-crx)插件，这些翻译插件内置了国内外专业翻译引擎：谷歌翻译、DeepL、必应翻译、百度翻译、腾讯翻译君、有道翻译、彩云小译、搜狗翻译、阿里翻译等。

以及知名翻译词典。

## 软件特点：

+ 支持主流PDF阅读器以及浏览器
+ 低延迟
+ 低功耗
+ 秒翻译
+ 支持：Windows、MacOS

软件免费，欢迎大家下载使用～

# 1. 安装

+ 软件下载地址：[下载饺子翻译助手](https://study.zwjjiaozhu.top/jz-translator/download.html)
+ 最新版本：1.4.2

## Windows：

+ 双击运行`饺子翻译助手.exe`安装程序.

+ 一路按默认选项安装即可.

+ 安装完毕后，会在桌面和开始菜单中有快捷方式，点击图标即可启动软件.

> 如果中途遇到杀毒软件拦截，正常现象，软件已通过：腾讯哈勃查毒，请放心安全问题.
>
> 解决：关闭这些安全软件或者将饺子翻译助手添加到安全软件的白名单中

## Mac：

+ 双击运行`饺子翻译助手.dmg`安装程序.

+ 然后将==Jz翻译助手==拖到应用程序中，即可完成安装

![安装界面](https://user-images.githubusercontent.com/34623422/187385140-ee961cbf-2855-4aa9-90fe-6f5ad6a780ae.png)

+ 安装完毕后，到启动台中启动软件，如果打开出现`应用程序已损坏`等字样，说明软件签名那一关没过.
+ **解决办法**：到安装包（可看上图）中的"修复已损坏问题"中按照里面的教程，一招就可搞定，这里就不多赘述.

# 2. 安装翻译插件

## 2.1 安装沙拉查词插件

其他翻译插件也是以下的步骤，原理一样

下载地址：见[首页下载入口](https://study.zwjjiaozhu.top/jz-translator/download.html)

**安装教程**（基于谷歌浏览器，）：

+ 直接将插件拖到安装界面，即可完成安装
+ 具体可看下方图片所示

![安装沙拉查词插件](https://user-images.githubusercontent.com/34623422/187385408-57604d3e-e2ea-4ec0-bb9a-d244290b4159.png)

## 2.1 配置沙拉查词快捷键

 

**默认快捷键：**

+ Mac下是`command+shift+T`
+ Windows下是`ctrl+shift+T`
+ 当然你也可以自定义，见[软件配置教程](https://study.zwjjiaozhu.top/jz-translator/config.html)（必看⚠️）

**设置快捷键：**

1. 进入谷歌浏览器的快捷键设置页面
   + 入口： [chrome://extensions/shortcuts](chrome://extensions/shortcuts)

2. 找到沙拉查词选项
3. 找到==“在独立窗口中搜索剪贴板内容”==
4. 设置快捷键（上面的默认快捷键，或者自定义的），按键盘键即可
5. 并将作用域改成`全局`
6. PS：具体可看下方图片所示

![配置沙拉查词快捷键](https://user-images.githubusercontent.com/34623422/187385889-88560fcd-bfae-4a8e-8b3d-ae3c790b2431.png)

## 2.3 开启读取剪贴板权限

沙拉查词的设置步骤：进入到插件设置界面 ->>左侧 `权限管理` ->>开启 `读取剪贴板`

其他翻译插件：一般是“权限”字样的入口中改成允许**读取剪贴板**功能，或者到设置页面 `Ctrl + F`搜索剪贴等字样.

## 沙拉查词的小知识

1. 自动发音，步骤：->>左侧 `发音设置` ->>开启 `英文自动发音`

2. 关闭每次划词时窗口获取焦点的功能，步骤：->>左侧 `快捷查询` ->>关闭 `出现时获取焦点`

# 视频教程：
[安装教程](https://www.bilibili.com/video/BV1D14y1e7d1/)
# 3. 使用

日常使用的时候，当鼠标光标处于![文本指针](https://user-images.githubusercontent.com/34623422/187386080-8e50c922-3484-4780-a540-7c11d961bdc4.png)状态（文本选择器、插入模式、文本指针)

+ 可以**双击**选中文本，然后翻译.
+ 也可以点击鼠标左键不放，**左右拖动**选中文字，松开鼠标，即可进行翻译.

❓那我可以在任意光标状态进行划词吗？

可以的，到这里[设置 CursorType 选项](https://study.zwjjiaozhu.top/jz-translator/config.html#编辑配置)，但我不建议.

## Windows：

双击运行后，即可划词使用

### 给软件添加必要的权限：

+ 辅助功能权限：
  + `系统偏好设置` ->> `安全性及隐私` ->> `辅助功能` ->> 将Jz翻译助手，添加进去
    ![添加辅助功能权限](https://user-images.githubusercontent.com/34623422/187386321-eff34248-cb31-4ec7-b65a-dc32afb234ed.png)

+ ~~输入监听权限~~（==1.4.0以前的版本需要==）：
  + `系统偏好设置` ->> `安全性及隐私` ->> `输入监听` ->> 将Jz翻译助手，添加进去
    ![添加输入监听权限](https://user-images.githubusercontent.com/34623422/187386388-4246ab78-79fd-4228-b589-8d538424f123.png)

+ 重启软件后就可正常运行

## 最后

软件免费，欢迎大家下载使用～

**Comment by zwjjiaozhu** [comment]
视频版安装教程：[https://www.bilibili.com/video/BV1D14y1e7d1/](https://www.bilibili.com/video/BV1D14y1e7d1/)
演示视频可见B站：[https://www.bilibili.com/video/BV1vt4y1g7ZG/](https://www.bilibili.com/video/BV1vt4y1g7ZG/)

**Comment by crimx** [comment]
Wow 功能很多的工具，请问会开源嘛

**Reply by zwjjiaozhu** [reply]
大佬居然回复了😁，代码写的烂就先不献丑了，未来会开源
